### ISSUE TITLE
Images not generated

### ISSUE THREAD

**Issue by pwim** [tokyodev-inc/tokyodev-devographics#269]
Social images when sharing / "Get image" is not generated.

**Comment by SachaG** [comment]
Yes, I'll do them all once we've finalized the contents. It's still not dynamic yet, sadly…

**Comment by pwim** [comment]
Results are finalized so we should do this.

**Comment by SachaG** [comment]
I'd rather wait for the japanese content to be finalized to do them all at once, if that's ok?
